docs/UserGuide.md

```diff
@@ -10,12 +10,12 @@ ATAS is a **desktop app for managing students’ particulars, optimized for use
 * [Quick start](#qs)
 * [Features](#features)
   * [Viewing help](#help) (coming soon)
-  * [Adding a students](#add)
-  * [Listing all students](#list)
-  * [Locating students by name](#find)
-  * [Deleting a student](#delete)
-  * [Editing a student's particulars](#edit)
-  * [Clearing all entries](#clear)
+  * [Adding a students](#addstu)
+  * [Listing all students](#liststu)
+  * [Locating students by name](#findstu)
+  * [Deleting a student](#deletestu)
+  * [Editing a student's particulars](#editstu)
+  * [Clearing all entries](#clearstu)
   * [Switching between tabs](#switch)
   * [Exiting the program](#exit)
   * [User confirmation prompt](#ucp) (coming soon)
```

Comment:
Perhaps we might want to remove "(coming soon)"?

Reply:
Good spot!